templates/legal/motd.html

```diff
@@ -0,0 +1,127 @@
+{% extends "legal/_base_legal.html" %}
+
+{% block title %}MOTD Information{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1pe_72kC7bClPqlgHU6maSMHYhAAuQdfCcNvAo-lDYVg/{% endblock meta_copydoc %}
+
+{% block content %}
+
+<div class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1 id="title">Legal Notice &mdash; MOTD Information</h1>
+      <p><strong>Version &mdash; June 2020</strong></p>
+      <p>This legal notice tells you about the information we collect using MOTD &mdash; Message of the Day. &ldquo;motd-news&rdquo; is a package that makes a call periodically to Canonical servers to get updated news for support and informational purposes. An example of an MOTD is shown at the end of this notice. As part of sending the message information is also collected by Canonical as set out below.</p>
+      <h2 id="who-are-we">Who are we?</h2>
+      <p>We are Canonical Group Limited. Our address is 5th Floor Blue Fin Building, 110 Southwark Street, London, SE1 0SU. You can contact us by post at the above address or by email to <a href="mailto:legal@canonical.com">legal@canonical.com</a> for the attention of &ldquo;Legal&rdquo;.</p>
+      <h2 id="what-information-do-we-collect">What information do we collect?</h2>
+      <p>The following data is being collected by Canonical</p>
+      <table>
+        <thead>
+          <tr>
+            <th>What is being sent?</th>
+            <th>Example</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Version of Ubuntu (/etc/lsb-release)</td>
+            <td>&ldquo;Ubuntu/18.04.3/LTS&rdquo;</td>
+          </tr>
+          <tr>
+            <td>Operating System Name (uname -o)</td>
+            <td>&ldquo;GNU/Linux&rdquo;</td>
+          </tr>
+          <tr>
+            <td>Kernel Version (uname -r)</td>
+            <td>&ldquo;4.15.0-72-generic&rdquo;</td>
+          </tr>
+          <tr>
+            <td>Machine Hardware Name (uname -m)</td>
+            <td>&ldquo;x86_64&rdquo;</td>
+          </tr>
+          <tr>
+            <td>CPU Brand & Model (/proc/cpuinfo)</td>
+            <td>&ldquo;Intel(R)/Core(TM)/i5-8500B/CPU/@/3.00GHZ&rdquo;</td>
+          </tr>
+          <tr>
+            <td>System Uptime (time since last boot) & Idle time (/proc/uptime)</td>
+            <td>&ldquo;uptime/108266.13/212047.71&rdquo;</td>
+          </tr>
+          <tr>
+            <td>cloud_id (/usr/bin/cloud-id) &mdash; Way to know what cloud (if any) this machine is running on.</td>
+            <td>&ldquo;aws&rdquo; or &ldquo;openstack&rdquo; or &ldquo;unknown&rdquo;</td>
+          </tr>
+          <tr>
+            <td>&ldquo;Curl&rdquo; version &mdash; the command used to get the message from the URL has it&rsquo;s version appended. This is standard practice in a &lsquo;user-agent&rsquo; string for HTTP requests.</td>
```

Comment:
"it's version appended" should be "its version appended".

Reply:
Thanks, fixed.